config/manifests/kustomization.yaml

```diff
@@ -2,7 +2,7 @@
 # used to generate the 'manifests/' directory in a bundle.
 resources:
 - bases/netobserv-operator.clusterserviceversion.yaml
-- ../default
+- ../openshift
```

Comment:
What are exactly the specificities with openshift?
The operator is published on operatorhub upstream, it is not openshift-only.
Upstream, there are two "variants" of operatorhub that w publish to:
- For openshift upstream: https://github.com/redhat-openshift-ecosystem/community-operators-prod
- Non-openshift: https://github.com/k8s-operatorhub/community-operators

Does it mean we need to produce 2 different bundles, one for each?

(And then there's also the mid/downstream build which is yet another thing)

Reply:
for openshift we have to place certain annotations on the service, crd for TLS authentications they are all in patch.yaml file under openshift folder, today the operator assume ovnkubernetes from what we have seen with KinD deployment so I don't understand how none openshift will work ? but if that is the case we can either generalize what I call kind in this diffs and if we findout its not good enough we can create new different configs as needed

Reply:
Ok, I think we need to generalize what you named kind then :)
To clarify, there is no dependency on ovk-kubernetes - at least, not anymore. Initially the operator was configuring ovn-k's IPFIX export, so there was that dependency indeed, however that was before we introduced the ebpf agent. That agent is pretty much agnostic to the underlying CNI, so it removed the dependency on ovn-k. We keep the "legacy" IPFIX mode, so that's why you will still find in code a reconciler related to ovn-k, but that's more of a corner case now.

Reply:
done moved to kubernetes folder